### PR TITLE
TravisCI setup

### PR DESCRIPTION
**Pull request by ganeshbsub**

### CODE REVIEW

**Comment by ganeshbsub** [comment]
@marcoow Do we also need to run [html-proofer](https://rubygems.org/gems/html-proofer/versions/2.6.4) ?

**Comment by marcoow** [comment]
yes, please

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: ruby
+rvm:
+- 2.1
+
+before_script:
+ - chmod +x ./script/cibuild # or do this locally and commit
```
Comment (marcoow) [inline .travis.yml]: Can you run this locally once and commit? Then we can remove the `before_script` step.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+script: ./script/cibuild
+
+env:
+  global:
+  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
+
+sudo: false # route your build to the container-based infrastructure for a faster build
+
+notifications:
+  email: false
```
Comment (marcoow) [inline .travis.yml]: missing newline at EOF

File: script/cibuild [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
+bundle exec jekyll build
```
Comment (marcoow) [inline script/cibuild]: missing newline at EOF

**Comment by ganeshbsub** [comment]
@marcoow Ready for merge. Please review. Should I squash the commits?

File: _config.yml [hunk 4]
```diff
@@ -31,10 +31,13 @@ title: simplabs
 author: simplabs GmbH
 description: simplabs builds solutions for the web and mobile devices
 
-url: simplabs.com
+url: http://www.simplabs.com
```
Comment (marcoow) [inline _config.yml]: this should be `http://simplabs.com` (no www)

File: blog/_posts/2013-06-15-authentication-in-emberjs.md [hunk 5]
```diff
@@ -83,7 +83,7 @@ $.cookie('auth_account', App.Auth.get('accountId'));
 
 #### Making authenticated requests
 
-The next step is to actually send the authentication token to the server. As the only point point of interaction between client and server in an ember.js app is **when the store adapter reads or writes data, the token has to be integrated in that adapter somehow**. As there’s not (yet) any out-off-the-box support for authentication in the [DS.RESTAdapter](https://github.com/emberjs/data/blob/master/packages/ember-data/lib/adapters/rest_adapter.js), I simply added it myself:
+The next step is to actually send the authentication token to the server. As the only point point of interaction between client and server in an ember.js app is **when the store adapter reads or writes data, the token has to be integrated in that adapter somehow**. As there’s not (yet) any out-off-the-box support for authentication in the DS.RESTAdapter, I simply added it myself:
```
Comment (marcoow) [inline blog/_posts/2013-06-15-authentication-in-emberjs.md]: this should now link to the new location of the `RestAdapter` - https://github.com/emberjs/data/blob/master/addon/adapters/rest.js

File: blog/_posts/2013-08-08-better-authentication-in-emberjs.md [hunk 6]
```diff
@@ -7,15 +7,15 @@ github-handle: marcoow
...
+When we started our first [ember.js](http://emberjs.com) project in June 2013, one of the first things we implemented was authentication. Now, almost 2 months later, **it has become clear that our initial approach was not really the best and had some shortcomings. So I implemented a better authentication** (mostly based on the embercasts on authentication).
 
 <!--break-->
 
 _I’m using the latest (as of early August 2013) [ember.js](http://emberjs.com) and [handlebars](http://handlebarsjs.com) releases in this example._
 
-_**Update:**I changed the section on actually using the token to use [$.ajaxPrefilter](http://api.jquery.com/jQuery.ajaxPrefilter/) instead of a custom ember-data adapter (thanks to [Marc for the comment](http://log.simplabs.com/post/57702291669/better-authentication-in-ember-js#comment-1035275502 "Marc's comment on the topic")!)_
+_**Update:**I changed the section on actually using the token to use [$.ajaxPrefilter](http://api.jquery.com/jQuery.ajaxPrefilter/) instead of a custom ember-data adapter (thanks to Marc for the comment!)_
```
Comment (marcoow) [inline blog/_posts/2013-08-08-better-authentication-in-emberjs.md]: please remove `(thanks to Marc for the comment!)` as that doesn't really make sense without the link

**Pull request merged**